ssz-rs/src/de.rs

```diff
@@ -66,7 +65,7 @@ pub trait Deserialize {
 
 fn deserialize_fixed_homogeneous_composite<T>(encoding: &[u8]) -> Result<Vec<T>, DeserializeError>
 where
-    T: SimpleSerialize,
+    T: Deserialize + crate::Sized,
```

Comment:
minor, but let's just import `Sized` at the top and drop the ref to `crate` here

Reply:
It is conflicting the builtin `Sized` here https://github.com/ralexstokes/ssz-rs/blob/main/ssz-rs/src/de.rs#L64. Nvm, I renamed it.